oci/go.mod

```diff
@@ -13,14 +13,12 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.4.0 // indirect
 	github.com/spf13/cobra v0.0.5
-	github.com/tektoncd/pipeline v0.10.1
+	github.com/tektoncd/pipeline v0.11.2
```

Comment:
Any reason for it to be `v0.11.2` instead of a later version ?

Reply:
That's the version when I first wrote this modification to the experimental tool  👀

Reply:
ok, let's get this in anyway, we can bump this in a follow-up :wink: